content/articles/indexers-in-c#/index (1).md

```diff
@@ -0,0 +1,281 @@
+### Indexers in c#
+### Introduction
+The concept of indexers is analogous to that of properties. Indexers and properties are quite similar in terms of linguistic qualities. Indexers make it possible to have indexed properties, which are properties that are referred to by one or more arguments. Those arguments are used to create an index into a set of values. Indexers are most commonly used in types that have the primary goal of encapsulating an internal collection or array.

```

Comment:
```suggestion
---
layout: engineering-education
status: publish
published: false
url: /indexers-in-c#/
title: Indexers in C#
description: The aim of this tutorial is to help the reader understand the concept of indexers in C#.
author: ezra-mukono
date: 
topics: [Languages]
excerpt_separator: <!--more-->
images:

  - url: /engineering-education/indexers-in-c#/hero.jpg
    alt: Indexers Example Image
---

The concept of indexers is analogous to that of properties. Indexers and properties are quite similar in terms of linguistic qualities. Indexers make it possible to have indexed properties, which are properties that are referred to by one or more arguments. Those arguments are used to create an index into a set of values. Indexers are most commonly used in types that have the primary goal of encapsulating an internal collection or array.
<!--more-->
```

Reply:
Hey @mukono10 had made some changes ...you were to resolve the requested changes and commit them..what you have done are not the changes I requested for..when you're done let me know any challenges let me know